### PR TITLE
Adds trace service tags for appeals controllers

### PR DESCRIPTION
**Pull request by patrickvinograd**
## Summary
Adds the newly-created Traceable concern to API controllers for the appeals-related services in vets-api, and tags their services so error traces will be associated correctly in Datadog.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/66141

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Requested Feedback
I updated CODEOWNERS because I don't think LH banana-peels owns these controllers in the main rails app. Feedback welcome on that front, or if anything assigned to Benefits-Team-1 should be for decision-reviews-be or vice-versa.

### CODE REVIEW

**Review by ericboehs — APPROVED** [review]

**Comment by rmtolmach** [comment]
@patrickvinograd To get the **Validate changes to DataDog Service Catalog Files** check passing, please update your branch with the latest changes from `master`. Thanks!

File: app/controllers/v1/supplemental_claims/contestable_issues_controller.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 module V1
   module SupplementalClaims
     class ContestableIssuesController < AppealsBaseControllerV1
+      service_tag 'appeal-application'
```
Comment (anniebtran) [inline app/controllers/v1/supplemental_claims/contestable_issues_controller.rb:6]: not sure if there's a rhyme or reason to the tags being assigned here, but this one feels a bit more vague than the other ones. any reason not to use `supplemental-claim` here (and below)?

Comment (patrickvinograd) [reply app/controllers/v1/supplemental_claims/contestable_issues_controller.rb:6]: I had that question too, but benefits crew POs picked that name - it does have some alignment in context with other `*-application` service names, indicating "this is a veteran applying for a benefit", vs. viewing information.

Comment (anniebtran) [reply app/controllers/v1/supplemental_claims/contestable_issues_controller.rb:6]: gotcha, makes sense. thanks for clarifying!

**Review by kristen-brown — APPROVED** [review]

**Pull request merged**